src/components/header/header.js

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+import { Icon } from 'react-fa';
+import './header.scss';
+import Logo from '../../components/logo/logo';
+import SlackLogo from '../../images/Slack-Monochrome-White.svg';
+
+export default class Hello extends React.Component {
+  constructor() {
+    super();
+  }
+
+  render() {
+    return (
+      <div className="row-fluid">
+        <div id="header" className="col-xs-12">
+          <div className="col-xs-2 logo">
```

Comment:
Just as a matter of simplifying things: any div that's acting as a grid piece to bootstrap should _only_ be doing that. I'd recommend nesting this if you need extra styles for `.logo`. Keeping track of overwriting styles gets messy, and you definitely want to avoid overwriting bootstrap's grid styles.